orca-core/src/main/groovy/com/netflix/spinnaker/orca/batch/adapters/TaskTasklet.groovy

```diff
@@ -143,19 +152,26 @@ class TaskTasklet implements Tasklet {
     return result
   }
 
-  private RepeatStatus cancel(Stage stage) {
+  private RepeatStatus cancel(Stage stage, boolean adjustStageStatusAndTasks = true) {
+    doCancel(stage, adjustStageStatusAndTasks)
```

Comment:
Really just so I could override it with an assertion in a test and not worry about the batch-specific `RepeatStatus` response